config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "service-name": "saf-lambda-function",
+    "bucket-input-folder": "unprocessed/",
+    "bucket-output-folder": "processed/",
+    "output-enabled": true,
+    "output-file-ext": ".json",
+    "output-clarifier": "_output" 
```

Comment:
I put the bucket name and the SAF CLI command string as environment variables because they have to be set by the user and may contain usernames and passwords (ex: splunk command). I put anything else configurable in this config file because the user could use the defaults set and I don't think any of this should contain sensitive information.

Let me know if this is appropriate or if any values should be configured differently. 

Also, if anyone has a better name for "output-clarifier". It's what I am using to change the name of the output file. It's referenced in the readme, but isn't a great name.

Reply:
How about ```output-suffix``` or ```output-prefix``` if that is what it does?